benchmarks/distributed/rpc/parameter_server/servers/AverageParameterServer.py

```diff
@@ -0,0 +1,142 @@
+import threading
+
+import torch
+import torch.distributed.rpc as rpc
+from utils import sparse_rpc_format_to_tensor, sparse_tensor_to_rpc_format
+
+from .ParameterServerBase import ParameterServerBase
+
+
+class AverageParameterServer(ParameterServerBase):
+
+    def __init__(
+        self,
+        rank,
+        trainer_count,
+        use_cuda_rpc
+    ):
+        r"""
+        a parameter server that averages the gradients
+        from trainers for each training iteration step.
+        gradients are added as they are received from trainers.
```

Comment:
nit: capital letter in the beginning of a sentence.

Reply:
okay.